packages/store/src/utils/selector-utils.ts

```diff
@@ -7,9 +7,33 @@ import {
   getStoreMetadata,
   SelectorMetaDataModel,
   SharedSelectorOptions,
-  StateClass
+  StateClass,
+  globalSelectorOptions
 } from '../internal/internals';
 
+const SELECTOR_OPTIONS_META_KEY = 'NGXS_SELECTOR_OPTIONS_META';
+
+export const selectorOptionsMetaAccessor = {
+  getFrom: (target: any): SharedSelectorOptions => {
+    return (target && (<any>target)[SELECTOR_OPTIONS_META_KEY]) || {};
+  },
+  setFor: (target: any, options: SharedSelectorOptions) => {
```

Comment:
rename `getFrom` to `getOptions`
rename `setFor` to `defineOptions`

Reply:
Done